.github/workflows/scorecard.yml

```diff
@@ -24,12 +25,12 @@ jobs:
 
     steps:
       - name: "Checkout code"
-        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # pin@v4
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # pin@v4
```

Comment:
The hash has updated here, but the pin comment remains the same—is this still v4?

Reply:
yes. There's some info as to how this works in [PROCEDURES.md](https://github.com/open-quantum-safe/liboqs/blob/main/docs/PROCEDURES.md#github-actions)